Makefile

```diff
@@ -99,6 +99,10 @@ LDFLAGS := \
 	-X k8s.io/component-base/version.gitMinor=${KUBE_MINOR_VERSION} \
 	-X k8s.io/component-base/version.buildDate=${BUILD_DATE} \
 	-extldflags '-static'
+
+# So the recursive Space MAkefile can use the same flags
```

Comment:
The wording of this comment suggests a dependency that runs in the wrong direction (the Space Makefile relying on a service provided by this Makefile).  I think an accurate and less alarming explanation is that this is a shorthand for explicitly setting the values of these variables in the nested `make` invocations.

Reply:
Agree. Will be moving to independent MAkefile